tests/test_omxplayer.py

```diff
@@ -1,19 +1,26 @@
 import unittest
 import os
+import sys
 import signal
 import dbus
 
-from nose_parameterized import parameterized
+from parameterized import parameterized
 from mock import patch, Mock, call, mock_open
 
 from omxplayer.dbus_connection import DBusConnectionError
 from omxplayer.player import OMXPlayer
 
+if sys.version_info[0] == 2:
+    builtin = '__builtin__'
```

Comment:
This logic is duplicated repeatedly in the test files, it'd be good to isolate it